### PR TITLE
update edaniels->viamrobotics

### PR DESCRIPTION
**Pull request by bazile-clyde**
The RDK no longer uses [edaniels/gostream](https://github.com/edaniels/gostream) since it was forked into viamrobotics at [viamrobotics/gostream](https://github.com/viamrobotics/gostream). See https://github.com/viamrobotics/rdk/commit/3e8408bdc8f79d1b0efc7ce63266821da4a80687

### CODE REVIEW

**Comment by nicksanford** [comment]
Hey! @bazile-clyde This PR doesn't pass lint or tests. Mind fixing it?

File: go.mod [hunk 1]
```diff
@@ -70,15 +70,14 @@ require (
 	github.com/desertbit/timer v0.0.0-20180107155436-c41aec40b27f // indirect
 	github.com/disintegration/imaging v1.6.2 // indirect
 	github.com/docker/go-units v0.5.0 // indirect
-	github.com/edaniels/gostream v0.0.0-20230424213557-e12afcfabcd8 // indirect
```
Comment (bazile-clyde) [inline go.mod]: Updating the RDK removed this indirect dependency which was the problem

**Comment by bazile-clyde** [comment]
@nicksanford done!

**Review by nicksanford — APPROVED** [review]
LGTM! Thanks for migrating to the new repo!

**Pull request merged**